Mixpanel/Mixpanel.m

```diff
@@ -765,7 +867,7 @@ - (void)startFlushTimer
 {
     [self stopFlushTimer];
     dispatch_async(dispatch_get_main_queue(), ^{
-        if (self.flushInterval > 0) {
+        if (self.flushInterval > 0 && ![self hasOptedOutTracking]) {
```

Comment:
if the user opts in later during the session, this change will prevent events to be sent and that's no bueno